packages/amplify_auth_cognito/lib/amplify_auth_cognito.dart

```diff
@@ -86,6 +86,11 @@ class AmplifyAuthCognito extends AuthPluginInterface {
     return res; 
   }
 
+  Future<AuthUser> getCurrentUser({AuthUserRequest request}) async {
+    final res = await _instance.getCurrentUser();
```

Comment:
Do you need the `res` or can you just do like `return (await _instance.getCurrentUser());` or some such?

Reply:
You could return directly, but I prefer this way for debugging purposes.